### ISSUE TITLE
__revit__.Application.Documents

### ISSUE THREAD

**Issue by GoogleCodeExporter** [architecture-building-systems/revitpythonshell#24]
```
What steps will reproduce the problem?
1. accessing anything after __revit__.Application.Documents.xxx
2.
3.

What is the expected output? What do you see instead?
Unexpected error, crash Revit

What version of the product are you using? On what operating system?
Revit 2015, ironpython 2.7.4, win 7 64bit,

Please provide any additional information below.
```

Original issue reported on code.google.com by `hao...@gmail.com` on 22 Aug 2014 at 4:59

**Comment by daren-thomas** [comment]
this issue seems to be related to the code completion running in a separate thread to the Revit API. this will be tricky to fix...

**Comment by daren-thomas** [comment]
fixed in eb5161d

**Comment by daren-thomas** [comment]
oops... only fixed for the interactive (modal) shell - still needs to be fixed for non-modal shell...

**Comment by daren-thomas** [comment]
fixed for non-modal shell in 83332e8
